docs/openstack_backend.rst

```diff
@@ -63,6 +63,10 @@ Usage
     $ openstack keypair create myself > myself.pem
     $ chmod 600 myself.pem
 
+* Setup the teuthology node::
+
+    $ teuthology-openstack --key-filename myself.pem --key-name myself --setup
+
```

Comment:
This new bullet point actually belongs in the previous section (`Setup`).

Reply:
Done